packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -531,7 +533,7 @@ export class VerticalBarChartBase
       : this._createColors()(1);
 
     // there might be no y value of the line for the hovered bar. so we need to check this condition
-    if (this._isHavingLine && selectedPoint[0].lineData?.y !== undefined) {
+    if (this._isHavingLine && selectedPoint[0].lineData?.y !== undefined && this._legendHighlighted(lineLegendText)) {
```

Comment:
>this._legendHighlighted(lineLegendText) [](http://example.com/codeflow?start=76&length=39)

this condition should be true if no legend highlighted as well

Reply:
done